packages/client/src/store/forum/thunk.ts

```diff
@@ -0,0 +1,27 @@
+import { createAsyncThunk } from '@reduxjs/toolkit'
+import {
+    CreateCommentDTO,
+    CreateTopicDTO,
+    TopicsGetParams,
+} from '@api/forum/types'
+import { forumApi } from '@api'
+
+export const getTopics = createAsyncThunk(
+    'forum/getTopics',
+    async (data: TopicsGetParams) => await forumApi.getTopics(data)
+)
+
+export const getTopic = createAsyncThunk(
+    'forum/getTopic',
+    async (id: string) => await forumApi.getTopic(id)
```

Comment:
Тут везде можно сократить
```suggestion
    forumApi.getTopic
```